### PR TITLE
cleaning up `docs` directory

### LINKED ISSUE #758: clean up /docs
It's confusing that /docs does not in fact contain docs.
- [ ] Add a readme that links to the actual docs
- [ ] Clean up whatever is unnecessary in the directory

### PR DESCRIPTION
**Pull request by pdg744**
Resolves #758

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
-  <a href="https://risczero.com/#gh-dark-mode-only"><img src="docs/assets/RISC_Zero_Logo_darkmode.png" height="100"></a>
-  <a href="https://risczero.com/#gh-light-mode-only"><img src="docs/assets/RISC_Zero_Logo_lightmode.png" height="100"></a>
+  <a href="https://risczero.com/#gh-dark-mode-only"><img src="assets/RISC_Zero_Logo_darkmode.png" height="100"></a>
+  <a href="https://risczero.com/#gh-light-mode-only"><img src="assets/RISC_Zero_Logo_lightmode.png" height="100"></a>
```
Comment (tzerrell) [inline README.md:3]: I think we should keep the logo assets in the docs folder, for 2 reasons:
* These are used for docs (namely, the main readme)
* It reduces the number of top-level folders

**Review by tzerrell — COMMENTED** [review]
Thank you!

**Comment by tzerrell** [comment]
Put up a PR to this branch that I think should fix the CI issue: https://github.com/risc0/risc0/pull/885

**Comment by pdg744** [comment]
I won't be on github much next week; I'm gonna pass this back to @tzerrell to take over the line

**Review by tzerrell — APPROVED** [review]
This is looking good to me, but the CI issues (which are happening to PRs generally and should be unrelated to this) will need to be resolved before this can land.

**Pull request merged**